# Generated for NaoQi method ALInfrared.send8
# Send 1 octet by IR.

# octet to send through IR.
int32 Octet
---


